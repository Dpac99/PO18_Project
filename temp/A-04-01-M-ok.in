101010
3
2
123456789
0
0
