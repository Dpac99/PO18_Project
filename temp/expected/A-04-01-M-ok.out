Identificador: Menu principal
1 - Abrir
2 - Guardar
3 - Portal Pessoal
0 - Sair
Escolha uma opção: Portal Pessoal
1 - Mostrar pessoa
2 - Alterar número de telefone
3 - Mostrar pessoas
4 - Procurar pessoa
0 - Sair
Escolha uma opção: Número de telefone: FUNCIONÁRIO|101010|123456789|José Carlos
Portal Pessoal
1 - Mostrar pessoa
2 - Alterar número de telefone
3 - Mostrar pessoas
4 - Procurar pessoa
0 - Sair
Escolha uma opção: Menu principal
1 - Abrir
2 - Guardar
3 - Portal Pessoal
0 - Sair
Escolha uma opção: 